cruise-control-metrics-reporter/src/main/java/com/linkedin/kafka/cruisecontrol/metricsreporter/CruiseControlMetricsReporter.java

```diff
@@ -116,7 +116,7 @@ public void close() {
   static String getBootstrapServers(Map<String, ?> configs) {
     Object port = configs.get(KafkaConfig.PortProp());
     String listeners = String.valueOf(configs.get(KafkaConfig.ListenersProp()));
-    if (listeners != null && listeners.length() != 0) {
+    if (!listeners.equals("null") && listeners.length() != 0) {
```

Comment:
Since this change is not backward compatible in a way that it does not handle the `listeners == null` case. So, is  `listeners` guaranteed never to be null?

Reply:
Correct, `String.valueOf(...)` does not return `null`, but can return `"null"`

Reply:
I see. Thanks